docs/projects/template-fps/setup-and-installing.md

```diff
@@ -1 +1,27 @@
-// TODO: Populate this page from content from: https://github.com/improbable/ECS-FPS/blob/master/docs/setup-and-installing.md
+# Set up the FPS Starter Project
+
+1. Follow the [machine setup guide]({{urlRoot}}/setup-and-installing#set-up-your-machine).
+2. Clone the `GDK-for-Unity-FPS-Starter-Project` repository:
+
+    |     |     |
+    | --- | --- |
+    | SSH | `git clone git@github.com:spatialos/GDK-for-Unity-FPS-Starter-Project.git` |
+    | HTTPS | `git clone https://github.com/spatialos/GDK-for-Unity-FPS-Starter-Project.git` |
+3. Setup dependencies by either:
+    - Running the bundled scripts: `scripts/powershell/setup.ps1` (Windows) or `scripts/shell/setup.sh` (Mac).
+    - Manually by following the instructions below.
+
+<%(#Expandable title="Manually setup dependencies")%>
+1. Clone the `GDK-for-Unity` repository alongside the `GDK-for-Unity-FPS-Starter-Project` so that they sit side-by-side:
```

Comment:
Can we maybe explain the `side-by-side` part in more detail since it can be kinda magical. Something that might help explaining this could be a diagrams for example:
```
<Common parent folder>
├── GDK-for-Unity-FPS-Starter-Project
├── GDK-for-Unity
```

Reply:
Yeah good shout